### PR TITLE
Feature/user lock composition

### PR DESCRIPTION
**Pull request by agualis**
# Description

Includes locked veBal in "My pool share". 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?

Given an account with locked veBal go to this pool: 

https://feature-user-lock-compositio.balancer-d6h.pages.dev/#/ethereum/pool/0x5c6ee304399dbdb9c8ef030ab642b10820db8f56000200000000000000000014

and check the composition table includes the locked fiat in its calculations (see example below).

## Visual context

Screenshot 2023-02-07 at 19 19 39

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] If package-lock.json has changes, it was intentional.
- [ ] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/composables/useUserPoolPercentage.ts [hunk 1]
```diff
@@ -1,19 +1,33 @@
...
   const { balanceFor } = useTokens();
   const { stakedShares } = usePoolStaking();
+  // Avoid lock queries when pool is not veBAL:
+  const { lock, lockedFiatTotal } = useLock({
+    enabled: isVeBalPool(pool.value.id),
+  });
   const { fNum2 } = useNumbers();
 
+  const lockedAmount = computed(() => {
+    // return lock.value?.lockedAmount || '0';
```
Comment (agualis) [inline src/composables/useUserPoolPercentage.ts:20]: @garethfuller I started using lockedAmount but I think that I should use `lockedFiatTotal` instead, which is not a fiat, so probably better renaming it to `lockedBalanceTotal` or `lockedAmountTotal`. Am I wrong?

**Comment by agualis** [comment]
@timjrobinson Could you check that the calculations looks good in the preview? (I used your account to test it but I could be missing something).

https://feature-user-lock-compositio.balancer-d6h.pages.dev/#/ethereum/pool/0x5c6ee304399dbdb9c8ef030ab642b10820db8f56000200000000000000000014

File: src/composables/useUserPoolPercentage.ts (hunk 1 above)
Comment (agualis) [reply src/composables/useUserPoolPercentage.ts:20]: Update: I saw that you already renamed it in another PR.

**Review by garethfuller — CHANGES_REQUESTED** [review]
Seems package-lock has been committed with changes when it shouldn't have been?

**Review by garethfuller — APPROVED** [review]
lgtm!

**Pull request merged**